### DISCUSSION TITLE
[#991930/983795] Chat Text obscures the Menu bar

### DISCUSSION THREAD

**Discussion by SIsilicon** [Mojang/minecraft-editor#43] [Bugs]
When performing operations like undo, it shows in chat. The problem however is that it starts from very top left corner of the Minecraft window, which is also where the menu bar is.
![image](https://user-images.githubusercontent.com/34734122/223885409-05a9826f-bf12-4375-ab37-2fc45dfe9c10.png)
I recommend moving it inside the player's view window instead or adjusting it to not overlap with the main menus.

**Comment by Landy005** [comment]
This also happens when coordinates are turned on, it currently blocks the Menu bar. Since there are coordinates in the bottom left corner the extra coordinates shouldn't show up at all!
![coordinates](https://user-images.githubusercontent.com/16018643/223905127-3e35eac0-aaa9-4fb7-a0fe-d31211e1f7fd.png)

**Reply by SIsilicon** [reply]
Your coordinates also show in the bottom left

**Comment by minecraftchloe** [comment]
Related to #65 - updated title to reflect internal tracking ID

**Comment by minecraftchloe** [comment]
Some more info: We've seen this bug as part of a broader investigation on DX11 and DX12 graphics cards. Do you happen to know which graphics card you're running?

**Reply by SIsilicon** [reply]
Intel celeron Integrated graphics. It supports DirectX 12 but Minecraft uses DirectX 11.

**Comment by minecraftchloe** [comment]
Should be fixed in current build or our next release this week -- lmk if this pops up again as an issue!!

**Reply by conmaster2112** [reply]
So its now invisible that chat messages? or its intended to be under editor UI?

**Reply by minecraftchloe** [reply]
It _should_ be within the viewport if the toggle to show coordinates is on, not overlapping on the Editor UI action bar. Let me know what you see in the current Preview, though!

![image (62)](https://github.com/Mojang/minecraft-editor/assets/56832421/14da249d-fc38-4429-b8bf-de906af32ad6)

**Reply by conmaster2112** [reply]
Awesome! Thank you so much
![image](https://github.com/Mojang/minecraft-editor/assets/64863757/6f08aed0-e73f-4896-bda8-a7afba5f7221)
